perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_attribute_def_def_mailTest.java

```diff
@@ -0,0 +1,55 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import com.google.common.collect.Lists;
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.Member;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests attribute module.
+ */
+public class urn_perun_member_attribute_def_def_mailTest {
+
+	private urn_perun_member_attribute_def_def_mail classInstance;
+	private PerunSessionImpl session;
```

Comment:
Please initialize all variables in all tests. now it works correctly even without the initialization, but when someone change the check method it may throw NullPointerException.

Reply:
Okay, variables initialized.